### ISSUE TITLE
Columns Add-on Error

### ISSUE THREAD

**Issue by douwe-ddyn** [FrDH/jQuery.mmenu#773]
Hi, 

Let me quickly use this opportunity to thank you for creating such an awesome plugin.

Using the columns addon with an 'on-canvas' menu, results in the following error message:

*this.$pnls is undefined - originating from jquery.mmenu.columns.js*

```javascript
$("#my-menu").mmenu({
    offCanvas: false,
    columns: {
        add: true,
        visible: { min: 1, max: 2 }
    }
});
```
jQuery.mmenu - 6.1.2
Firefox Developer Edition - 55.0b12 (64-bit)

Please let me know if you require any additional information. Thanks in advance.

**Comment by FrDH** [comment]
Hi, 
Thank you for bringing this to my attention, I'll investigate later on.

**Comment by FrDH** [comment]
I found the issue and will fix it in the next release.

**Comment by douwe-ddyn** [comment]
Awesome! Thanks again.

**Comment by FrDH** [comment]
Should be fixed in 6.1.3
